.github/workflows/requirements.txt

```diff
@@ -1,4 +1,4 @@
-lsst-daf-butler
+lsst-daf-butler-26.2024.200
```

Comment:
Yes it is unused; we are installing lsst-daf-butler via the eups install in the workflow. I can git rm this requirements.txt file, since it keeps confusing me into adding dependencies to it.